### PR TITLE
Update solution.md

### PR DESCRIPTION
**Pull request by ericregnier**
Updated correct behavior on pac solution import with specifying when SolutionPackageType is Both

### CODE REVIEW

File: power-platform/developer/cli/reference/solution.md [hunk 1]
```diff
@@ -435,7 +435,7 @@ Max asynchronous wait time in minutes. The default value is 60 minutes.
 
 #### `--path` `-p`
 
-Path to solution zip file. If not specified, assumes the current folder is a cdsproj project.
+Path to solution zip file. If not specified, assumes the current folder is a cdsproj project. If `SolutionPackageType` in cdsproj file is set to `Both`, the managed solution is used by default.
```
Comment (JimDaly) [inline power-platform/developer/cli/reference/solution.md:438]: This can't be edited manually. It will be overwritten next time docs are generated.

**Review by JimDaly — CHANGES_REQUESTED** [review]
See my other comments.

@petrochuk & @laneswenka 
You may want to just add this to the /help content for the import solution command.

**Comment by ericregnier** [comment]
> Updated correct behavior on pac solution import with specifying when SolutionPackageType is Both

@JimDaly thanks and noted. I submitted another PR remark file instead. Cheers!

**Comment by JimDaly** [comment]
Closing this in favor of #2685

**Pull request closed without merging**